public_html/php/Widar.php

```diff
@@ -44,8 +44,8 @@ public function __construct ( /*string*/ $toolname = '', $cookie_lifetime = null
 			);
 			$this->oa = new MW_OAuth ( WbstackMagnusOauth::getOauthParams(
 				'widar',
-				'/tools/widar',
-			), $cookie_lifetime ) ;
+				'/tools/widar'
+			), '', '', '', $cookie_lifetime ) ;
```

Comment:
So do I understand correctly that we still need to now made a change to cradle (but not quickstatements since it doesn't use Widar) here https://github.com/wbstack/cradle/blob/92e59b0997aa94cc08c90cf0e79edbbb29ffd18c/public_html/api.php#L33 to head towards the desired outcome of the ticket?

Reply:
Why? If you do not pass any of these parameters, the previous behavior should be left as is as everything that is added is guarded by a `if ($this->cookie_lifetime !== null)`. If you don't pass the old param, it will do what was done before.

Reply:
The caller (in this case `wbstack/widar`) will need to pass a value still, that is correct.